test.tf

```diff
@@ -182,15 +287,14 @@ resource "checkly_check_group" "test-group1" {
     }
 
     basic_auth {
-      username = "user"
-      password = "pass"
+      username = "maxmustermann"
```

Comment:
While we're looking at usage examples, I think we should probably be careful of encouraging people to put secrets in Terraform code. If we make this a variable reference, then you could require the relevant secret to be set in the environment, for example, or (better) a git-crypted locals file.

Reply:
will add an example with a note